docs/development/core/development-functional-tests.asciidoc

```diff
@@ -277,47 +314,50 @@ await testSubjects.click(‘containerButton’);
+* Helpers for retrying operations
+* Popular methods:
+** `retry.try(fn, onFailureBlock)` - execute `fn` in a loop until it succeeds or the default try timeout elapses, optional `onFailureBlock` is executed before each retry attempt 
+** `retry.tryForTime(ms, fn, onFailureBlock)` execute `fn` in a loop until it succeeds or `ms` milliseconds elapses, , optional `onFailureBlock` is executed before each retry attempt
+
 **kibanaServer:**:::
-* Source: {blob}test/functional/services/kibana_server/kibana_server.js[test/functional/services/kibana_server/kibana_server.js]
+* Source: {blob}test/common/services/kibana_server/kibana_server.js[test/common/services/kibana_server/kibana_server.js]
 * Helpers for interacting with Kibana's server
 * Commonly used methods:
 ** `kibanaServer.uiSettings.update()`
 ** `kibanaServer.version.get()`
 ** `kibanaServer.status.getOverallState()`
 
 **esArchiver:**:::
-* Source: {blob}test/functional/services/es_archiver.js[test/functional/services/es_archiver.js]
+* Source: {blob}test/common/services/es_archiver.ts[test/common/services/es_archiver.ts]
 * Load/unload archives created with the `esArchiver`
 * Popular methods:
 ** `esArchiver.load(name)`
 ** `esArchiver.loadIfNeeded(name)`
 ** `esArchiver.unload(name)`
 
-**docTable:**:::
-* Source: {blob}test/functional/services/doc_table.js[test/functional/services/doc_table.js]
-* Helpers for interacting with doc tables
+Full list of services that are used in functional tests can be found here: {blob}test/functional/services[test/functional/services]
 
-**pointSeriesVis:**:::
-* Source: {blob}test/functional/services/point_series_vis.js[test/functional/services/point_series_vis.js]
-* Helpers for interacting with point series visualizations
 
 **Low-level utilities:**:::
 * es
-** Source: {blob}test/functional/services/es.js[test/functional/services/es.js]
+** Source: {blob}test/common/services/es.ts[test/common/services/es.ts]
 ** Elasticsearch client
 ** Higher level options: `kibanaServer.uiSettings` or `esArchiver`
 * remote
-** Source: {blob}test/functional/services/remote/remote.js[test/functional/services/remote/remote.js]
-** Instance of https://theintern.github.io/leadfoot/module-leadfoot_Command.html[Leadfoot's `Command]` class
+** Source: {blob}test/functional/services/remote/remote.ts[test/functional/services/remote/remote.ts]
+** Instance of https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html[WebDriver] class
 ** Responsible for all communication with the browser
-** Higher level options: `testSubjects`, `find`, and `PageObjects.common`
-** See the https://theintern.github.io/leadfoot/module-leadfoot_Command.html[leadfoot/Command API] for full API
+** To perform browser actions use `remote` service 
+** For searching and manipulating with DOM elements use `testSubjects` and `find` services
+** See the https://seleniumhq.github.io/selenium/docs/api/javascript/[selenium-webdriver docs] for full API
```

Comment:
I would change this to:

See the https://seleniumhq.github.io/selenium/docs/api/javascript/[selenium-webdriver docs] for the full API.